product-scenarios/pom.xml

```diff
@@ -74,22 +74,36 @@
         <activemq.client.version>5.15.8</activemq.client.version>
         <pax.logging.api.version>1.10.1</pax.logging.api.version>
     </properties>
-
-    <modules>
-        <module>scenarios-commons</module>
-        <module>1-integrating-systems-that-communicate-in-heterogeneous-message-formats</module>
-        <module>2-Bridging-systems-that-communicate-in-different-protocols</module>
-        <module>3-connecting-web-apis-cloud-services</module>
-        <module>4-gateway</module>
-        <module>5-route-messages-between-systems</module>
-        <module>7-connecting-with-packaged-applications</module>
-        <module>8-connect-devices-to-enterprise</module>
-        <!--TODO Scenario 10 is commented until VFS is supported-->
-        <!--<module>10-file-processing</module>-->
-        <!--<module>11-Asynchronous-message-processing</module>-->
-        <module>13-micro-services</module>
-        <module>14-periodically-execute-an-integration-process</module>
-    </modules>
+    <profiles>
+        <profile>
+            <id>profile_general</id>
+            <modules>
+                <module>scenarios-commons</module>
+                <module>1-integrating-systems-that-communicate-in-heterogeneous-message-formats</module>
+                <module>2-Bridging-systems-that-communicate-in-different-protocols</module>
+                <module>3-connecting-web-apis-cloud-services</module>
+                <module>4-gateway</module>
+                <module>5-route-messages-between-systems</module>
+                <module>7-connecting-with-packaged-applications</module>
+                <module>8-connect-devices-to-enterprise</module>
+                <!--TODO Scenario 10 is commented until VFS is supported-->
+                <!--<module>10-file-processing</module>-->
+                <!--<module>11-Asynchronous-message-processing</module>-->
+                <module>13-micro-services</module>
+                <module>14-periodically-execute-an-integration-process</module>
+            </modules>
+        </profile>
+        <profile>
+            <id>profile_artifacts</id>
+            <modules>
+                <module>scenarios-commons</module>
+                <module>1-integrating-systems-that-communicate-in-heterogeneous-message-formats</module>
+                <module>2-Bridging-systems-that-communicate-in-different-protocols</module>
```

Comment:
Don't we need all the modules test in profile_general, here?

Reply:
All the profiles tested in the master are tested here
https://github.com/wso2/product-ei/blob/e369ddc9bde30c5c2ff67df7e5ee0647e192697c/product-scenarios/pom.xml#L78